examples/ONE/ONE.ino

```diff
@@ -1235,6 +1235,7 @@ static void createMqttTask(void) {
             if (agMqtt.publish(topic.c_str(), syncData.c_str(),
                                syncData.length())) {
               Serial.println("MQTT sync success");
+              resetWatchdog();
```

Comment:
I propose to centralize the watchdog reset feature and feed in information such as e.g. "MQTT publishing worked" and then let it decide on criteria such as if e.g. posting to the AG backend is enabled if it should reset or not. @pnt325 can you look into this?

Reply:
I'll check it. 

On newest version I have callback direct to `ag->watchdog.reset()` for reset.

Reply:
@pnt325 What is the status here. Has this been addressed and can I close this PR?

Reply:
> @pnt325 What is the status here. Has this been addressed and can I close this PR?

The watchdog reset by post data to AG success or offline mode. It's not reset by MQTT publish. We don't have any MQTT connect to AG. The MQTT connection by user. I think we can close it